Simplenote/Classes/FileManager+Simplenote.swift

```diff
@@ -34,4 +34,17 @@ extension FileManager {
 
         return targetURL
     }
+    
+    /// Writes a data file to the Documents Folder
+    ///
+    class func writeDataToDocuments(data: Data, to targetURL: URL) -> URL? {
```

Comment:
- Since this API doesn't really save `to documents` exclusively (the URL could point anywhere), perhaps we could drop the `DataToDocuments` suffix?

- Since the code isn't related to FileManager itself (and the log mentions Note Exporter), maybe the best spot for it is `PDFLinkActivityItemProvider` itself?